contracts/src/v0.8/dev/functions/Functions.sol

```diff
@@ -21,10 +21,17 @@ library Functions {
     // In future version we may add other languages
   }
 
+  enum AggregationMethod {
+    Mode,
+    Median
+    // In future version we may add other methods (e.g. Mean, Custom)
+  }
+
   struct Request {
     Location codeLocation;
     Location secretsLocation;
     CodeLanguage language;
+    AggregationMethod aggregationMethod;
```

Comment:
Does the order matter? Will it break current setup if added in the middle? Ideally we won't have a deployment dependency so if adding a new field at the end helps then let's do that.

Reply:
I don't believe it will because `Functions.Request` is never used in storage, only in memory while building the request.

Reply:
What I'm worried about is _ethabidecodelog_ task in the pipeline.

Reply:
Gotcha, I'll double check once I have a chance to re-visit this PR